project1/templates/sucess.html

```diff
@@ -0,0 +1,12 @@
+<html>
+<body>
+    <h2> your data</h2>
+    <h3>Registered Mail-id:{{mail}}<br>Name:{{username}}<h3>
+    <h4>The following is the list of users:</h4>
+    <ul>
+        {%for i in name: %}
```

Comment:
Good job on writing loops. However, it is not required to show all previous users to the user who is logging in.